cmd/plume/release.go

```diff
@@ -555,3 +592,34 @@ func modifyReleaseMetadataIndex(spec *fcosChannelSpec, commitId string) {
 		plog.Fatalf("uploading release metadata json: %v", err)
 	}
 }
+
+func compareStaticReleaseInfo(a, b BuildMetadata) bool {
+	if a.Version != b.Version || a.Endpoint != b.Endpoint {
+		return false
+	}
+	return true
+}
+
+// returns -1 if a is a subset of b, 0 if equal, 1 if a is not a subset of b
+func compareCommits(a, b []Commit) int {
+	if len(a) > len(b) {
```

Comment:
I can see it being superfluous but I don't think it's inherently wrong. If a is larger than b it cannot be a subset so this exists as an early exit hatch rather than checking the individual entries.

Reply:
Don't mind me, it's hot today. I misread that as an equality check. The actual code is correct, my brain a bit less :)